wrapper/desktop/CMakeLists.txt

```diff
@@ -31,4 +31,6 @@ INCLUDE("${CMAKE_CURRENT_LIST_DIR}/../../CMakeLists.txt")
 
 ADD_EXECUTABLE(${OPENMAPPER_APP} ${OM_APP_RESOURCES})
 
-TARGET_LINK_LIBRARIES(${OPENMAPPER_APP} debug ${OPENMAPPER_LIB} optimized ${OPENMAPPER_LIB})
+TARGET_LINK_LIBRARIES(${OPENMAPPER_APP}
+  debug ${OPENGL_LIBRARIES} optimized ${OPENGL_LIBRARIES}
```

Comment:
I don’t know if we still need the desktop stuff here... eventually remove in the next pr